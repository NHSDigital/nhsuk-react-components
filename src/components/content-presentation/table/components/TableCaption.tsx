import React, { FC, HTMLProps } from 'react';
import classNames from 'classnames';

const TableCaption: FC<HTMLProps<HTMLTableCaptionElement>> = ({ className, ...rest }) => (
  <caption className={classNames('nhsuk-table__caption', className)} {...rest} />
);
TableCaption.displayName = 'Table.Caption';

export default TableCaption;
