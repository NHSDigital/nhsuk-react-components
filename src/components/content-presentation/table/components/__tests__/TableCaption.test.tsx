import React from 'react';
import { render } from '@testing-library/react';
import TableCaption from '../TableCaption';

describe('TableCaption', () => {
  it('matches snapshot', () => {
    const { container } = render(<TableCaption />);

    expect(container).toMatchSnapshot();
  });
});
