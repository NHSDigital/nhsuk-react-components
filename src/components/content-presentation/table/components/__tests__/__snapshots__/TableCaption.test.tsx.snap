// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableCaption matches snapshot 1`] = `
<div>
  <caption
    class="nhsuk-table__caption"
  />
</div>
`;
